### ISSUE TITLE
Is itget quote volume of BTC/USDT from Binance by OHLC？

### ISSUE THREAD

**Issue by youlchat** [ccxt/ccxt#24476]
### Programming Languages

Python

### Code

```
  

```

**Comment by youlchat** [comment]
![Weixin Screenshot_20241207103807](https://github.com/user-attachments/assets/4e483974-a60f-414a-88b9-463ddd4b4bfe)

**Comment by sc0Vu** [comment]
@youlchat You can call `publicGetKlines` to retrieve quote volume directly.

**Comment by youlchat** [comment]
> @youlchat You can call `publicGetKlines` to retrieve quote volume directly.

Thank you very much！ Are your family name is 赖?  真有缘

**Comment by gbelkin** [comment]
Why not just add the quote volume field to parse_ohlcv function?
For both ccxt and async_support

I would gladly open a PR for it. 

``      inverse = self.safe_bool(market, 'inverse')
        volumeIndex = 7 if inverse else 5
        quoteVolumeIndex = 5 if inverse else 7
        return [
            self.safe_integer_2(ohlcv, 0, 'openTime'),
            self.safe_number_2(ohlcv, 1, 'open'),
            self.safe_number_2(ohlcv, 2, 'high'),
            self.safe_number_2(ohlcv, 3, 'low'),
            self.safe_number_2(ohlcv, 4, 'close'),
            self.safe_number_2(ohlcv, volumeIndex, 'volume'),
            self.safe_number_2(ohlcv, quoteVolumeIndex, 'quoteVolume'),
        ]``

**Comment by carlotestor** [comment]
Yes. The unified `fetch_ohlcv` OHLCV array is fixed at `[timestamp, open, high, low, close, volume]` where `volume` is base volume — quote volume isn't part of the unified structure. Two ways to get it on Binance:

**From the raw response** (index 7 is quote asset volume):

```python
klines = exchange.publicGetKlines({'symbol': 'BTCUSDT', 'interval': '1d', 'limit': 1000})
df = pd.DataFrame(klines, columns=[
    'timestamp','open','high','low','close','volume','close_time',
    'quote_volume','trades','taker_base','taker_quote','ignore'])
```

For futures use `fapiPublicGetKlines` with the same layout.

**Or compute it** — for daily candles `close * volume` is a fine approximation, but the exact figure is the exchange's index-7 value.

Note your DataFrame constructor is what's failing: you pass 7 column names but `fetch_ohlcv` returns 6 elements per row. Drop `'quote_volume'` there, or switch to the implicit call above.

@gbelkin — a PR extending `parse_ohlcv` isn't likely to be accepted since it would break the fixed-length unified OHLCV shape shared across all exchanges and languages; the implicit-API route is the supported way.
